src/prune/scene/world_scene.cpp

```
@@ -33,6 +99,43 @@ namespace prune {
         render_overlay(renderer);
     }
 
+    void WorldScene::begin_runtime()
+    {
+        capture_authored_objects();
+    }
+
+    void WorldScene::reset_runtime()
+    {
+        const std::vector<GameObjectId> selected_ids(
```

Clang (14.0.6)

[warning] variable 'selected_ids' is not initialized

(cppcoreguidelines-init-variables)